ci/Dockerfile

```diff
@@ -3,17 +3,12 @@
 
 FROM ubuntu:focal
 
-# Increase this to force a rebuild of the Docker image (in Cirrus, in particular).
```

Comment:
I'd leave this in. We won't need it for Spicy anymore but could still want to update dependencies.